src/features/navigation/SearchStackNavigator/routes.ts

```diff
@@ -15,7 +13,7 @@ const initialSearchStackRouteName: SearchStackRouteName = 'SearchLanding'
 const routes: SearchStackRoute[] = [
   {
     name: 'SearchLanding',
```

Comment:
on pourrait utiliser l'enum aussi ici (à faire que si on choisi de le faire dans l'autre)

```suggestion
    name: SearchView.Landing,
```